core/src/main/java/nl/nn/adapterframework/pipes/XmlIf.java

```diff
@@ -184,7 +184,7 @@ public void setRegex(String regex){
 	}
 
 	/**
-	 * specifies the version of xslt to use
+	 * specifies the version of xslt to use. This pipe supports up to and including XSLT version 3.0
```

Comment:
Perhaps the doc for version here should be same as in other classes that have an XsltVersion argument?